addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4518,6 +4518,7 @@ registry.sizing = SnippetOptionWidget.extend({
             ev.preventDefault();
             self.options.wysiwyg.odooEditor.automaticStepUnactive('resizing');
             isMobile = weUtils.isMobileView(self.$target[0]);
+            let resizeResolve;
```

Comment:
Why do you need to define this this far from the initialization? Can't you declare this right above the first mutex lock?